src/kernels/MIOpenTensorKernelsHip.cpp

```diff
@@ -103,21 +103,21 @@ extern "C" __global__ void Op1dTensorGeneric(const MIOPEN_TYPE* a,
 extern "C" __global__ void Op2dTensorGeneric(const MIOPEN_TYPE* a,
                                              const MIOPEN_TYPE* b,
                                              MIOPEN_TYPE* c,
-                                             const long Aoffset,
-                                             const long Boffset,
-                                             const long Coffset,
-                                             const uint32_t b_c,
-                                             const uint32_t c_c,
-                                             const uint32_t a_nstride,
-                                             const uint32_t a_cstride,
-                                             const uint32_t b_nstride,
-                                             const uint32_t b_cstride,
-                                             const uint32_t c_nstride,
-                                             const uint32_t c_cstride,
+                                             const DIM_TYPE Aoffset,
+                                             const DIM_TYPE Boffset,
+                                             const DIM_TYPE Coffset,
```

Comment:
I think it's better to keep the offsets 64 bit.

Reply:
I changed this for Op2dTensorGeneric kernel and at the same time for the last 2 PRs that I created for 64b support

https://github.com/ROCm/MIOpen/pull/3313
https://github.com/ROCm/MIOpen/pull/3317